doc/replication.md

```diff
@@ -236,9 +224,9 @@ if (replicator_push.getState() != Replicator.State.COMPLETE) {
 When using IndexManager for indexing and querying data, it needs to be updated after replication completes:
```

Comment:
Also change this while you are updating -- "querying data, we recommend you update after replication completes to avoid a wait for indexing to catch up when the new data is first queried:"